README.md

```diff
@@ -1,7 +1,145 @@
-# MRL
+# [Matryoshka Representations for Adaptive Deployment](https://arxiv.org/abs/2205.13147)
+Aditya Kusupati, Gantavya Bhatt, Aniket Rege, Matthew Wallingford, Aditya Sinha, Vivek Ramanujan, William Howard-Snyder, Kaifeng Chen, Sham Kakade, Prateek Jain, Ali Farhadi
 
-This project is primarily organized into 
-1. Training MRL models
-2. Evaluating MRL models via image retrieval, few shot learning
-3. Analysis of MRL models via superclass hierarchy and oracle performance
+This repository contains the code for training and evaluating Matryoshka Representation with CNN backbone, along with visualizations. Training code is built upon [FFCV](https://github.com/libffcv/ffcv-imagenet) modified for MRL. The repository is organized as follows:
```

Comment:
replace CNN with ResNet-50? We did not use any other backbone (ie mobilenet) in the paper right?